back/engines/free/email_campaigns/spec/models/email_campaigns/campaigns/project_phase_started_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe EmailCampaigns::Campaigns::ProjectPhaseStarted do
+  let(:campaign) { create(:project_phase_started_campaign) }
+
+  describe 'ProjectPhaseStarted Campaign default factory' do
+    it { expect(campaign).to be_valid }
+  end
+
+  describe '#campaign_enabled?' do
+    context 'when campaign is is enabled at campaign level' do
+      it 'returns false when campaign is disabled at phase level' do
+        phase = create(:phase, campaigns_settings: { project_phase_started: false })
+        notification = create(:project_phase_started, phase: phase)
+        notification_activity = create(:activity, item: notification, action: 'created')
+
+        expect(campaign.enabled).to be true
```

Comment:
This condition is put in the context, so maybe it makes sense to put the code there too. Like that

```ruby
context 'when campaign is is enabled at campaign level' do
  before { campaign.update!(enabled: true) }
```

Thi way, it'll be immediately clear what's the difference between two context blocks.